python/cudf/cudf/_libxx/issorted.pyx

```diff
@@ -0,0 +1,42 @@
+from cudf._libxx.cudf cimport *
+from cudf._libxx.cudf import *
+from cudf._libxx.utils cimport *
+from cudf._libxx.utils import *
+
+from cudf._libxx.includes.issorted cimport is_sorted
+
+def issorted(Table source_table, order=[], null_prec=[]):
```

Comment:
should `null_prec` be `null_order`? looks like both are used below, but not sure where `null_order` is coming from?